### DISCUSSION TITLE
Can't push my commit

### DISCUSSION THREAD

**Discussion by PoMalaW** [community/community#39587] [Projects and Issues] [answered]
### Select Topic Area

Question

### Body

Hello,

I'm not experimented a lot with git and github and I have a problem.

I can't push my project and I have this message when I do 
$ git push origin master 

Username for 'https://github.com': PoMalaW 
Password for 'https://PoMalaW@github.com': 
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/PoMalaW/Unity.git/'

I checked the link but I don't find the solution, can somebody help me ?

The url is set correctly, the password is good, I tried 3/4 times, I commited well so I don't understand

Thanks for your time and hope you will be able to help me ^^

**Comment by airtower-luna** [comment, marked as answer]
As the message says, you cannot use your password for `git push` or other Git operations (but you still need it to log in). You need to [create a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and enter that instead when Git asks for your password. When creating the token make sure to select the **repo** scope (for repository access, like pushing) and if you use Actions also **workflow** (required to modify workflow files).

**Reply by PoMalaW** [reply]
Thanks I will check that :D
